doc/pipeline-details.md

```diff
@@ -15,15 +15,14 @@ flowchart
     subgraph trigger_service[Trigger Service]
         trigger[Trigger] --> kernel_revision{New kernel <br/>revision ?}
         kernel_revision --> |No| trigger
```

Comment:
```
|No| trigger
```
What does `trigger` mean here?  I think it should be something more like `sleep`.

Reply:
Yes, will fix this.